<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>askForBoxesOptions</class>
 <widget class="QDialog" name="askForBoxesOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Options for Box mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="infoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>How many boxes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="boxesSpinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Default skip value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="skipSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Again Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="againSpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>-20</number>
       </property>
       <property name="maximum">
        <number>-1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hard Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hardSpinBox_2">
       <property name="minimum">
        <number>-20</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Good Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="goodSpinBox_3">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Easy Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="easySpinBox_4">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/sanki.qrc">
       <normaloff>:/icons/confirm.svg</normaloff>:/icons/confirm.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/sanki.qrc"/>
  <include location="../../../Resources/sanki.qrc"/>
 </resources>
 <connections/>
</ui>
